### ISSUE TITLE
Documentation could mention how to execute a test case

### ISSUE THREAD

**Issue by haesleinhuepf** [clEsperanto/CLIc#240]
On [this page](https://github.com/clEsperanto/CLIc/blob/master/docs/how_to_contribute/kernel_test_case.md#verify-that-the-test-is-working), it says "Execute it and make sure it runs without errors.". It would be great if it would explain how to run a test case.

**Comment by haesleinhuepf** [comment]
When I execute the built `test_....exe` files, they say something like:
```
C:\structure\code\CLIc\build\tests\tier7>test_voronoi_otsu_labeling.exe
Running main() from C:\structure\code\CLIc\build\_deps\googletest-src\googletest\src\gtest_main.cc
[==========] Running 0 tests from 0 test suites.
[==========] 0 tests from 0 test suites ran. (0 ms total)
[  PASSED  ] 0 tests.
```

**Comment by haesleinhuepf** [comment]
Partial solution:
```
C:\structure\code\CLIc\build>ctest -R test_voronoi_otsu_labeling
```

**Comment by StRigaud** [comment]
closed, fixed by #262

[how to run a test](https://clesperanto.github.io/CLIc/tests.html)
